Release 4.5.3 (#157)
blackbaud/skyux-popovers: CHANGELOG.md

# 4.5.3 (2021-07-08)

> - Fixed the dropdown component to close after clicking an item when items are nested in multiple layers, such as `*ngFor` and `*ngIf` directives. [#156](https://github.com/blackbaud/skyux-popovers/pull/156)

Reviewer: ```suggestion
- Fixed the dropdown menu component to close after users select items that are nested in multiple layers, such as `*ngFor` and `*ngIf` directives. [#156](https://github.com/blackbaud/skyux-popovers/pull/156)
```
I think we should add "menu" to be clear about which component we are talking about. The other edits are just trying to make the description of the change more clear because I had trouble processing the current version. But let me know if the changes aren't accurate or don't work for any other reason.
